Blackjack/Other/NSMutableArray_Shuffling.m

```diff
@@ -0,0 +1,23 @@
+//
+//  NSMutableArray_Shuffling.m
+//  Blackjack
+//
+//  Created by Alex on 29/09/2022.
+//
+
+#import "NSMutableArray_Shuffling.h"
+
+@implementation NSMutableArray (Shuffling)
+
+- (void)shuffle
+{
+    NSInteger count = [self count];
+    if (count <= 1) return;
```

Comment:
`if` is always in a new line, not same line

Reply:
Done